client/components/CandidateReview/CandidateTestPlanRun/InstructionsRenderer.jsx

```diff
@@ -168,9 +177,14 @@ const InstructionsRenderer = ({
     return (
         <>
             <NumberedList>{allInstructionsContent}</NumberedList>
+            {/* TODO: Remove 3 following lines to remove the Success Criteria once #863 is merged. Will need that
+                  functionality to show the commands and assertions specification table and if there are any
+                  exceptions, especially 0-level assertions */}
             <Heading>{pageContent.instructions.assertions.header}</Heading>
             {pageContent.instructions.assertions.description}
             <NumberedList>{assertionsContent}</NumberedList>
+            {settingsContent.length ? settingsContent : null}
+
```

Comment:
Are the changes in this file relevant to the PR? I recognize them and I wonder, is it possible they snuck in from one of your other PRs?

Reply:
+1 to this. Not sure if this is relevant to the PR.

Reply:
No. I don't think those changes are relevant to this PR. Should they be removed?

Reply:
Yes let's remove them. Thanks!